core/src/main/java/io/seata/core/rpc/netty/RmNettyRemotingClient.java

```diff
@@ -87,6 +87,38 @@ public void init() {
     private RmNettyRemotingClient(NettyClientConfig nettyClientConfig, EventExecutorGroup eventExecutorGroup,
                                   ThreadPoolExecutor messageExecutor) {
         super(nettyClientConfig, eventExecutorGroup, messageExecutor, TransactionRole.RMROLE);
+        // set enableClientBatchSendRequest
+        // New configuration takes precedence
+        String newConfig = ConfigurationFactory.getInstance().getConfig(ConfigurationKeys.ENABLE_RM_CLIENT_BATCH_SEND_REQUEST);
+        if(StringUtils.isNotBlank(newConfig)){
+            enableClientBatchSendRequest = Boolean.parseBoolean(newConfig);
+            ConfigurationCache.addConfigListener(ConfigurationKeys.ENABLE_RM_CLIENT_BATCH_SEND_REQUEST, new ConfigurationChangeListener() {
+                @Override
```

Comment:
the change of `enableclientbatchsendrequest` is not in the same thread

Reply:
It has been fixed